news/2022-05-20-beatmap-spotlights-season-6-spring-2022-high-speed.md

```diff
@@ -0,0 +1,283 @@
+
+Maybe that was spooky enough after all.
+
+*primarily suggested by [x\_angelkawaii\_x](https://osu.ppy.sh/users/566276)*
+
+*described by [Nelly](https://osu.ppy.sh/users/4741164)*
+
+### osu!mania
+
+#### Curators' Pick
+
+[Camellia - dreamless wanderer [flying in the void of deep-dream]](https://osu.ppy.sh/beatmapsets/1580929#mania/3228051), mapped by [paradoxus_](https://osu.ppy.sh/users/11185275)
+
+Rarely do we come across a map that provides bold concepts with nuanced gameplay elements to the table.
+
+dreamless wanderer is easily one of those charts with its integration of LN grace elements more commonly found in 4k and technical, reading intensive, patterning. This yields a combination which is quite a rarity even for unranked! While the highlights of the chart are found in the intense complextro bits, the intentionaly usage of pattern repetition and build really wraps the chart together.
+
+I definitely see this chart setting the groundwork for more expressive charting in the future for 7K ranked. Good job paradoxus_!
+
+*described and primarily suggested by [lepnai](https://osu.ppy.sh/users/5314573)*
+
+#### Players' Pick
+
+[Camellia - #1f1e33 [Future]](https://osu.ppy.sh/beatmapsets/1184620#mania/2485263), mapped by [Murumoo](https://osu.ppy.sh/users/8001433)
+
+This spectacle of a map #1f1e33 mapped by Murumoo really has a lot of extraordinary concepts and representations, but the Future diff is a must play for people who enjoy comfortable gameplay, which is everyone!
+
+The wubs and synths that are being represented in contrast to the hi hats, which create a perfect balance and make the map a lot more entertaining. But let me tell you, the kiai is by far one of the most important areas of a map, and it really does pull it off.
+
+This map will trip you up in excitement and pour a feeling of satisfaction when you see that pass on your screen, truly a wonder to behold.
+
+*described and primarily suggested by [[ Decku ]](https://osu.ppy.sh/users/13360768)*
+
+## Inside the Spotlights Team
+
+This season, we will say farawell to some of our members as well as welcome some fresh blood to the ranks of Spotlights Curators, thanks for all your hard efforts to deliver the seasonal doso of outstanding beatmap material!
+
+**Curator additions:**
+
+- **osu!**: DeviousPanda
+- **osu!taiko**: \[Zeth\], Nifty
+- **osu!catch**: Nelly
+- **osu!mania**: Antalf, Protastic101
+
+**Curator resignations:**
+
+- **osu!**: Ryuusei Aika
+- **osu!taiko**: Arrival, Nardoxyribonucleic
+- **osu!catch**: Snowless, x_angelkawaii_x
+- **osu!mania**: \[ Decku \], BringoBrango,  Kaito-kun
+
+Hydria will also be stepping away as the osu!mania curators' team leader after almost two years of service, CrumpetFiddler will be the one to ensure things go smooth on the mania side from now on.
+
+OnosakiHito will also say farawell to the osu!taiko curators' team after leading it since the very first season of modern Spotlights, Hivie will take over his tasks.
```

Comment:
```suggestion
OnosakiHito will also say farewell to the osu!taiko curators' team after leading it since the very first season of modern Spotlights, but worry not, because Hivie will take over his tasks.
```